docs/en/latest/admin-api.md

```diff
@@ -1497,6 +1497,102 @@ HTTP/1.1 200 OK
 {"key":"\/apisix\/secrets\/vault\/test2","value":{"id":"vault\/test2","token":"apisix","prefix":"apisix","update_time":1669625828,"create_time":1669625828,"uri":"http:\/\/xxx\/get"}}
 ```
 
+When `{secretmanager}` is `aws`:
```

Comment:
It would be sectionally more clear to use h4 title. e.g.

* change `When `{secretmanager}` is `vault`:` to `#### When Secret Manager is Vault`
* change `When `{secretmanager}` is `aws `:` to `#### When Secret Manager is AWS`
* change `When `{secretmanager}` is `gap `:` to `#### When Secret Manager is GCP`

Reply:
Resolved.